warehouse/integrations/deltalake/deltalake_test.go

```diff
@@ -379,6 +384,649 @@ func TestIntegration(t *testing.T) {
 			})
 		}
 	})
+
+	t.Run("Load Table", func(t *testing.T) {
```

Comment:
Keeping this as part of Integration tests instead of Unit Tests, since starting for the cluster/SQL warehouse takes around 10-15 mins.